src/extension.ts

```diff
@@ -75,6 +79,10 @@ export function activate(context: vscode.ExtensionContext) {
 
     context.subscriptions.push(runCommand);
     context.subscriptions.push(solidityWatcher);
+
+    item.text = "$(beaker) Aderyn Start";
```

Comment:
You can substitute "beaker" with any name available here https://github.com/microsoft/vscode-codicons/tree/main/src/icons